databricks/koalas/groupby.py

```diff
@@ -2104,7 +2104,10 @@ def nunique(self, dropna=True):
                 + F.when(F.count(F.when(col.isNull(), 1).otherwise(None)) >= 1, 1).otherwise(0)
             )
 
-        should_include_groupkeys = isinstance(self, DataFrameGroupBy)
+        # The behavior of GroupBy.nunique has been changed since pandas 1.1.0
+        should_include_groupkeys = False
+        if LooseVersion(pd.__version__) < LooseVersion("1.1.0"):
+            should_include_groupkeys = isinstance(self, DataFrameGroupBy)
```

Comment:
I don't think we need this anymore, and we can remove `should_include_groupkeys` parameter from `_reduce_for_stat_function` now.